content/applications/hr/recruitment/new_job.rst

```diff
@@ -442,7 +492,99 @@ Description
 When the applicant begins the survey, the text entered in this tab appears at the top of the survey
 page. Enter any information or descriptions that would be helpful to the applicant.
 
-End Message
+End message
 ~~~~~~~~~~~
 
 After the survey is complete, the message entered in this tab is displayed for the applicant.
+
+Additional options
+~~~~~~~~~~~~~~~~~~
+
+Once the interview form has been configured, there are several options available: :ref:`Share
+<share>`, :ref:`Create Live Session <live-session>`, :ref:`Test <test>`, and :ref:`Print <print>`
+
+.. _share:
+
+Share
+*****
+
+The interview form can be shared with applicants. Typically, this is done directly from the
+applicant form, but the survey can also be shared in this view.
+
+To send the form to a candidate, click the :guilabel:`Share` button in the top-left corner of the
+survey form. A :guilabel:`Share a Survey` pop-up window appears.
+
+First, select the :guilabel:`Recipients` using the drop-down menu. Multiple selections can be made
+in this field. The :guilabel:`Subject` field auto-populates with `Participate to (name of survey)`
+but can be modified, if desired.
+
+If any attachments need to be sent with the email, click the :guilabel:`Attachments` button. A file
+explorer pop-up window appears. Navigate to the file to add, then click :guilabel:`Open` to select
+it and attach it to the email.
+
+If there is a deadline for the survey to be completed, enter the date in the :guilabel:`Answer
+deadline` field. Click the field to the right of :guilabel:`Answer deadline` and a calendar pop-up
+window appears. Navigate to the desired date and click on the day to select it. Next, select the
+time using the two drop-down fields, one for the hours, one for the minutes. Finally, click
+:guilabel:`Apply` to select both the date and time.
+
+The email template being used is populated in the :guilabel:`Mail Template` field. The email
+template can be changed by selecting another template form the drop-down menu.
+
+.. note::
+   Changing the :guilabel:`Mail Template` does **not** change any other part of the email pop-up
+   window. Only the email body changes if another email template is selected.
+
+When the email is properly configured, click :guilabel:`Send` to send the interview.
+
+.. _live-session:
+
+Create live session
+*******************
+
+To create a live online session where participants can complete the interview form in real-time,
+click the :guilabel:`Create Live Session` button in the top-left of the survey form.
+
+The interview survey loads in a new tab. In this tab, several items appear:
+
+- :guilabel:`Attendees`: before anyone is logged in to take the interview survey, a
+  :guilabel:`Waiting for attendees...` box appears on the screen. As attendees login, the box is
```

Comment:
```suggestion
  :guilabel:`Waiting for attendees...` box appears on the screen. As attendees log in, the box is
```
Right?

Reply:
Yes! login and log in always trip me up.